ui/components/app/modals/update-account-nickname-modal/update-account-nickname-modal.container.js

```diff
@@ -0,0 +1,36 @@
+import { connect } from 'react-redux';
+import { hideModal } from '../../../../store/actions';
+import * as actions from '../../../../store/actions';
+import UpdateAccountNicknameModal from './update-account-nickname-modal.component';
+
+function mapStateToProps(state) {
```

Comment:
Our current pattern is to do all state selection using hooks within the functional component itself. We should do that for `addressBookEntry` as well

Reply:
Fixed.